.github/workflows/prerelease.yml

```diff
@@ -2,8 +2,6 @@ name: prerelease
 
 on:
   pull_request:
-    # closed will be triggered when a pull request is merged. This is to keep https://github.com/firebase/SpecsTesting up to date.
-    types: [closed]
```

Comment:
Why is this deleted?

Reply:
This is for testing purpose, since each test takes hours. Updating the condition will terminate the current testing run and the update will not trigger the test. Have pushed another commit to revert the triggering event of both prerelease and release workflow.